alibi/models/pytorch/tests/test_model.py

```diff
@@ -318,17 +299,15 @@ def test_compute_loss_multimodal(multimodal_model, dataset):
         'compile': True,
         'optimizer': optimizer_class,
         'loss': loss_fn,
-        'metrics': metric
+        'metrics': [AccuracyMetric()]
```

Comment:
Since you've changed these to be explicit, should we remove the global variables defined here? https://github.com/SeldonIO/alibi/blob/ea108699258c50284059da8f0d8c684590a0f2b0/alibi/models/pytorch/tests/test_model.py#L48-L49